frontend/src/components/pages/Default.tsx

```diff
@@ -18,6 +20,43 @@ const Button = ({ text, path }: ButtonProps) => {
   );
 };
 
+const TeamInfoDisplay = () => {
+  const { teamName, numTerms, members, isActive } = useContext(SampleContext);
+  return (
+    <div>
+      <h2>Team Info</h2>
+      <div>Name: {teamName}</div>
+      <div># terms: {numTerms}</div>
+      <div>
+        Members:{" "}
+        {members.map(
+          (name, i) => ` ${name}${i === members.length - 1 ? "" : ","}`,
+        )}
+      </div>
+      <div>Active: {isActive ? "Yes" : "No"}</div>
+    </div>
+  );
+};
+
+// Example: how to apply custom colors
+const DesignLibraryExamples = () => {
+  return (
+    <>
+      <VStack>
```

Comment:
can we move this to our new file called `ComponentLibrary.tsx`?

Reply:
oh +1 good catch